adagrams/game.py

```diff
@@ -1,11 +1,114 @@
+import random
+
 def draw_letters():
-    pass
+
+    letter_probability = {
```

Comment:
Totally fine to define this dictionary here, but if you hoist up to the module scope, Python won't need to recreate the dictionary every time `draw_letter` is called.